src/firebase/firebase.js

```diff
@@ -2,40 +2,14 @@ import firebase from 'firebase/app';
 import 'firebase/auth';
 import 'firebase/firestore';
 
-let apiKey;
-let authDomain;
-let databaseURL;
-let projectId;
-let storageBucket;
-let messagingSenderId;
-let appId;
-
-if (process.env.NODE_ENV !== 'production') {
-  apiKey = process.env.REACT_APP_FIREBASE_API_KEY;
-  authDomain = process.env.REACT_APP_FIREBASE_AUTH_DOMAIN;
-  databaseURL = process.env.REACT_APP_FIREBASE_DATABASE_URL;
-  projectId = process.env.REACT_APP_FIREBASE_PROJECT_ID;
-  storageBucket = process.env.REACT_APP_FIREBASE_STORAGE_BUCKET;
-  messagingSenderId = process.env.REACT_APP_FIREBASE_MESSAGING_SENDER_ID;
-  appId = process.env.REACT_APP_FIREBASE_APP_ID;
-} else {
-  apiKey = process.env.FIREBASE_API_KEY;
-  authDomain = process.env.FIREBASE_AUTH_DOMAIN;
-  databaseURL = process.env.FIREBASE_DATABASE_URL;
-  projectId = process.env.FIREBASE_PROJECT_ID;
-  storageBucket = process.env.FIREBASE_STORAGE_BUCKET;
-  messagingSenderId = process.env.FIREBASE_MESSAGING_SENDER_ID;
-  appId = process.env.FIREBASE_APP_ID;
-}
-
 const firebaseConfig = {
-  apiKey,
-  authDomain,
-  databaseURL,
-  projectId,
-  storageBucket,
-  messagingSenderId,
-  appId
+  apiKey: process.env.REACT_APP_FIREBASE_API_KEY,
```

Comment:
Will this work when we deploy, i thought the environment variables prefixed with `REACT_APP` were for local env variables only?

Reply:
By using React Scripts I don't see that the prefix is only used for a specific environment: https://create-react-app.dev/docs/adding-custom-environment-variables/
If you want to have different env vars depending on the env, you can have multiple files with the env as the extension. I.e. .env.development, .env.production, etc: https://create-react-app.dev/docs/adding-custom-environment-variables/#what-other-env-files-can-be-used

Reply:
Ah ok, yeah that makes sense, must have just been the examples i've seen they have used the `REACT_APP` ones for dev